src/main/java/org/example/homework4/LeapYear.java

```diff
@@ -0,0 +1,18 @@
+package org.example.homework4;
+
+public class LeapYear {
+    public static void main(String[] args) {
+        int year = 7;
+         if (leapYear(year))  {
+             System.out.println(year + " is a Leap Year");
+         } else System.out.println(year + " isn't a Leap Year or out of range [1;9999]");
+    }
+    private static boolean leapYear(int yearNumber) {
```

Comment:
In the task description the method should be `isLeapYear`
And it is the general naming convention for method which returns boolean (e.g isAvailable(), canBeUsed() etc...)